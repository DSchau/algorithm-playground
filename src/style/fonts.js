// @flow
export const SANS_SERIF = `'Montserrat', sans-serif`;
export const SERIF = `'Bitter', serif`;
