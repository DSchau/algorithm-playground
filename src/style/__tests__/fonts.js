import * as fonts from '../fonts';

test('it has expected fonts', () => {
  expect(Object.keys(fonts)).toMatchSnapshot();
});
