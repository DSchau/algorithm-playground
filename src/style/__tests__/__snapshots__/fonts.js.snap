// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it has expected fonts 1`] = `
Array [
  "SANS_SERIF",
  "SERIF",
]
`;
